common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_action.h

```diff
@@ -261,7 +257,13 @@ enum eActionOp_MONITOR: uint8_t {
     ACTION_MONITOR_CHANNEL_SCAN_RESULTS_NOTIFICATION = 0x41,
     ACTION_MONITOR_CHANNEL_SCAN_ABORT_NOTIFICATION = 0x42,
     ACTION_MONITOR_CHANNEL_SCAN_FINISHED_NOTIFICATION = 0x43,
-    ACTION_MONITOR_ENUM_END = 0x44,
+    ACTION_MONITOR_STEERING_CLIENT_SET_GROUP_REQUEST = 0x50,
+    ACTION_MONITOR_STEERING_CLIENT_SET_GROUP_RESPONSE = 0x51,
+    ACTION_MONITOR_STEERING_CLIENT_SET_REQUEST = 0x52,
+    ACTION_MONITOR_STEERING_CLIENT_SET_RESPONSE = 0x53,
+    ACTION_MONITOR_STEERING_EVENT_CLIENT_ACTIVITY_NOTIFICATION = 0x54,
+    ACTION_MONITOR_STEERING_EVENT_SNR_XING_NOTIFICATION = 0x55,
+    ACTION_MONITOR_ENUM_END = 0x56,
```

Comment:
Probably should mention in the commit message why you decided to reorder the opcodes...